src/pydvl/utils/parallel/backlog.py

```diff
@@ -0,0 +1,37 @@
+from copy import copy
+from functools import partial
+from typing import Callable, Generator, Generic, List, Tuple, TypeVar
+
+T = TypeVar("T")
+
+
+def wrap_index(*args, i: int, fn: Callable, **kwargs):
+    return i, fn(*args, **kwargs)
+
+
+class Backlog(Generic[T]):
```

Comment:
Please don't add this functionality to this PR. I would have a dozen comments to make and we don't have time if we want to include the seeding in 0.7.0. Revert this commit and make a separate PR

Reply:
But it isn't reproducible without. So in that case we have to move the test cases for reproducibility as well.

Reply:
Removed and put into https://github.com/aai-institute/pyDVL/pull/417 for further discussion.